### PR TITLE
Adds obsidian-plugin-update-tracker plugin

### PR DESCRIPTION
**Pull request by swar8080**
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/swar8080/obsidian-plugin-update-tracker

## Release Checklist
- [x] I have tested the plugin on
  - [ ]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [x]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

### CODE REVIEW

**Comment by swar8080** [comment]
Let me know if stepping through the code together on discord would help here, just in case the control flow and logic is hard to follow

**Comment by liamcain** [comment]
- [async onClose() {}](https://github.com/swar8080/obsidian-plugin-update-tracker/blob/f8d7723316ae002c7ee2bb6724ebc0b54269c2e8/src/main.tsx#L131-L132) I think you'll want to call `unmountComponentAtNode` to fully unmount your React component onClose
